docs/tutorials/markdown-guide.md

```diff
@@ -187,8 +232,44 @@ Another Text
 ***
 - - - -
 
-[More Information](https://www.markdownguide.org/basic-syntax/#horizontal-rules).
+### **Code**
+
+#### *Javascript*
```

Comment:
i believe code syntax highlighting is specific to
docusaurus, its ok to mention it here  just make a note that some markdown is github specific and another is docusaurus specific